modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -1,25 +1,47 @@
 # frozen_string_literal: true
 
-require 'evss/error_middleware'
-
 module ClaimsApi
   module V2
     module Veterans
       class ClaimsController < ClaimsApi::V2::ApplicationController
-        # TODO: REMOVE BEFORE IMPLEMENTATION
-        skip_before_action :authenticate, only: %i[index]
-        ICN_FOR_TEST_USER = '1012667145V762142'
-        # TODO: REMOVE BEFORE IMPLEMENTATION
-
         def index
-          raise ::Common::Exceptions::Unauthorized if request.headers['Authorization'].blank?
-          unless params[:veteranId] == ICN_FOR_TEST_USER
-            raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Resource not found')
-          end
+          veteran = build_veteran(veteran_id: params[:veteranId], loa: @current_user.loa)
+          detail  = 'Veteran not found'
+          raise ::Common::Exceptions::ResourceNotFound.new(detail: detail) unless veteran_found?(veteran: veteran)
```

Comment:
i went the route of never giving back a 404 if the Veteran is present or not. If the Veteran doesnt exist in MPI its still a 403:
https://github.com/department-of-veterans-affairs/vets-api/pull/7369/files#diff-9e276f2ddc002e7e8d30faaf090c0613cc94fe20f05e9353804d1210966fd9a2R10

I did this so that an person who is unauthorized couldnt determine whether a Veteran "existed" or not.

I think the "only 403" approach makes sense, but the bigger thing is just being consistent, so definitely up for discussing this if you disagree.

Reply:
Eh, personally it feels a little odd to return a 403 when something doesn't exist, but I understand the intention of obscuring that result.  But I agree, consistency is most important, so I updated the PR to follow the same pattern you introduced in [your PR](https://github.com/department-of-veterans-affairs/vets-api/pull/7369#pullrequestreview-704406712).